workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -25,6 +25,17 @@ internal static class DownloadCoreSdk
 
         private const string DownloadForceMenuItem = "SpatialOS/Download CoreSdk (force)";
 
+        private static readonly List<PluginDirectoryCompatibility> pluginsCompatibilityList = new List<PluginDirectoryCompatibility>
+        {
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/Android", false, new List<BuildTarget> { BuildTarget.Android }, null, false),
```

Comment:
Either leave the mobile folders out (to add them together with the rest of the mobile stuff) or add them to the Download program. Will be more consistent.

Reply:
Good point, didn't clean this up when rebasing